source/system/String.ooc

```diff
@@ -26,6 +26,7 @@ String: class {
 		this _buffer free()
 		super()
 	}
+	safeFree: func { if (!this _isLiteral) this free() } // Will not free literals automatically
```

Comment:
I'd rather not have to worry about if it's reused or not. I think this should be hidden inside the implementation details. Interface for string objects should be the same if object is a literal or not (something like we have for recyclable buffers now).